src/image/loading_displaying.js

```diff
@@ -1175,18 +1297,37 @@ p5.prototype.image = function(
     _sh
   );
 
-  // tint the image if there is a tint
-  this._renderer.image(
-    img,
-    vals.sx,
-    vals.sy,
-    vals.sw,
-    vals.sh,
-    vals.dx,
-    vals.dy,
-    vals.dw,
-    vals.dh
-  );
+  //if it is not graphics nor framebuffer but WEGL instance
+  //default to use 3D rendering
+  if (this._renderer instanceof p5.RendererGL && !isP5G && !isP5Fbo) {
```

Comment:
I feel like maybe this check should be a check for whether the `dz` parameter exists rather than the type of the image? since you could plausibly be drawing a graphic or a framebuffer both in 3D or without specifying a z